packages/shared/src/components/auth/AuthModal.tsx

```diff
@@ -16,7 +16,10 @@ import AnalyticsContext from '../../contexts/AnalyticsContext';
 export type AuthModalProps = { trigger?: AuthTriggersOrString } & ModalProps;
 
 const DiscardActionModal = dynamic(
-  () => import('../modals/DiscardActionModal'),
+  () =>
+    import(
+      /* webpackChunkName: "discardActionModal" */ '../modals/DiscardActionModal'
```

Comment:
Why dont we update webpack configuration to use camel case, and we can use the webpackChunkName for custom things

Reply:
Didn't know that was possible 🤯

Reply:
Ah didn't know that was an option, but it might prove difficult to sync between SWC (next) and webpack on the extension side?
Not sure if you have any idea about that?

Reply:
@el I had a read through but it seems you would still rely on manually naming them right?
See [this article](https://www.codemzy.com/blog/how-to-name-webpack-chunk), although would love to hear if you know another way?